python/lsst/meas/algorithms/__init__.py

```diff
@@ -24,13 +24,12 @@
 from .defects import *
 from .psfDeterminerRegistry import *
 from .starSelectorRegistry import *
-from .algorithmRegistry import *
 from .findCosmicRaysConfig import *
 from .detection import *
 from .gaussianPsfFactory import *
-from .measurement import *
 from .loadReferenceObjects import *
 from .replaceWithNoise import *
+from .loadReferenceObjects import *
```

Comment:
This line is a duplicate of the one two above it, probably from a rebase conflict.